lib/widget/camera.dart

```diff
@@ -38,32 +38,59 @@ class CameraScreen extends StatefulWidget with Invokable, HasController<MyCamera
   @override
   Map<String, Function> setters() {
     return {
-      'mode': (type) =>
-          _controller.mode = CameraMode.values.from(type) ?? CameraMode.both,
-      'initialCamera': (type) => _controller.initialCamera =
-          InitialCamera.values.from(type) ?? InitialCamera.back,
+      'mode': (value) => _controller.initCameraMode(value),
+      'initialCamera': (value) => _controller.initCameraOption(value),
       'useGallery': (value) => _controller.useGallery =
           Utils.optionalBool(value) ?? _controller.useGallery,
       'maxCount': (value) => _controller.maxCount =
           Utils.optionalInt(value) ?? _controller.maxCount,
       'preview': (value) => _controller.preview =
           Utils.optionalBool(value) ?? _controller.preview,
+      'errormessage': (value) =>
```

Comment:
@hamxa152 allow for expressions such as 'Max count must not be ${this.maxCount}' as well in the messages. @vusters please comment on which framework method to call to handle that